### PR TITLE
pd-ctl: add subcommand for checking cluster status

### PR DESCRIPTION
**Pull request by 5kbpers**
Signed-off-by: 5kbpers <tangminghua@pingcap.com>

### What problem does this PR solve?

We need a way to check the replication_mode status (close https://github.com/pingcap/pd/issues/2444)

### What is changed and how it works?
Add a subcommand `pd-ctl cluster status` for checking the status.

### Check List

Tests

- Unit test
- Manual test (add detailed scripts or steps below)
  * `pd-ctl cluster status`

### Release note

- Add a subcommand for checking cluster status

### CODE REVIEW

**Comment by rleungx** [comment]
Does the document also need to be updated?

File: tests/pdctl/cluster/cluster_test.go [hunk 1]
```diff
@@ -63,14 +66,24 @@ func (s *clusterTestSuite) TestClusterAndPing(c *C) {
 	echo := pdctl.GetEcho([]string{"-u", pdAddr, "--cacert=ca.pem", "cluster"})
 	c.Assert(strings.Contains(echo, "no such file or directory"), IsTrue)
 
-	// cluster status
-	args = []string{"-u", pdAddr, "cluster", "status"}
+	// cluster info
+	args = []string{"-u", pdAddr, "cluster"}
```
Comment (lhy1024) [inline tests/pdctl/cluster/cluster_test.go:70]: replaced the original command?

Comment (5kbpers) [reply tests/pdctl/cluster/cluster_test.go:70]: No, the test here used a wrong command before. 🤔

**Pull request merged**